src/directives/formio.js

```diff
@@ -38,6 +38,69 @@ module.exports = function() {
           $scope._src += 'live=1';
         }
 
+        // Build the display map.
+        $scope.show = {};
+        var boolean = {
+          'true': true,
+          'false': false
+        };
+
+        var submission = $scope._submission || {data: {}};
+        var updateComponents = function() {
+          // Change the visibility for the component with the given key
+          var updateVisiblity = function(key) {
+            $element
```

Comment:
Is this the best way to do this? We could set a variable on the element and ng-show="show[key]" or something like that.